### PR TITLE
make the aks show table output contain currentKubernetesVersion column

### PR DESCRIPTION
**Pull request by zjpjack**
This will add a new column "currentKubernetesVersion" for az aks show --output table.
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
aks

**Review by wangzelin007 — APPROVED** [review]
By the way if you want to release the new extension version, please write the description of changes into HISTORY.rst and update setup.py.

File: src/aks-preview/azext_aks_preview/_format.py [hunk 1]
```diff
@@ -88,6 +88,7 @@ def _aks_table_format(result):
         location: location,
         resourceGroup: resourceGroup,
         kubernetesVersion: kubernetesVersion,
+        currentKubernetesVersion: currentKubernetesVersion,
```
Comment (FumingZhang) [inline src/aks-preview/azext_aks_preview/_format.py:91]: Just curious, what's the difference between `kubernetesVersion` and `currentKubernetesVersion`?

**Review by FumingZhang — APPROVED** [review]

File: src/aks-preview/azext_aks_preview/_format.py (hunk 1 above)
Comment (zjpjack) [reply src/aks-preview/azext_aks_preview/_format.py:91]: For 20211101preview api, we have a new feature "Alias Minor Version", where customers can input "1.19" for kubernetesVersion. currentKubernetesVersion will be the exact kubernetesVersion we provision for them, like "1.19.7". If kubernetesVersion is a specific version, then both properties value are the same.

**Comment by zjpjack** [comment]
> By the way if you want to release the new extension version, please write the description of changes into HISTORY.rst and update setup.py.

**Comment by zjpjack** [comment]
> By the way if you want to release the new extension version, please write the description of changes into HISTORY.rst and update setup.py.

@wangzelin007 add description and updated setup.py

**Review by zhoxing-ms — APPROVED** [review]

**Pull request merged**